config/settings.yml

```diff
@@ -1037,7 +1037,6 @@
   :container_deployment_wizard: false
   :datawarehouse_manager: false
 :prototype:
```

Comment:
I left the prototype section here although it's empty because I think i will be useful next time someone has a prototype.